dalle2_pytorch/train_diffusion_prior.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import os
+from dalle2_pytorch import DiffusionPrior
+from embedding_reader import EmbeddingReader
+from dalle2_pytorch import DiffusionPriorNetwork
+import numpy as np
+import math
+from tqdm import tqdm
+import torch
+from torch import nn
+import wandb
+os.environ["WANDB_SILENT"] = "true"
+
+def train(image_embed_dim,
+          image_embed_url,
+          text_embed_url,
+          batch_size,
+          train_percent,
+          val_percent,test_percent,
+          num_epochs,
+          loss_type,
+          condition_on_text_encodings,
+          device,
+          learning_rate=0.01):
+    # DiffusionPriorNetwork 
+    prior_network = DiffusionPriorNetwork( dim = image_embed_dim, depth = 6, dim_head = 64, heads = 8).to(device)
```

Comment:
adding these model params to the function params (and then the cli) would be best, that way they would be easy to tweak and they would be reported to wandb